sdk/core/Azure.Core/tests/OperationInternalOfTTests.cs

```diff
@@ -511,7 +511,7 @@ public TestOperation(Func<bool, CancellationToken, ValueTask<OperationState<int>
             public ValueTask<OperationState<int>> UpdateStateAsync(bool async, CancellationToken cancellationToken) => _updateStateAsyncHandler(async, cancellationToken);
         }
 
-        private class CallCountStrategy : DelayStrategy
```

Comment:
Curious: would it make sense to validate that Operation will work with the new DelayStrategy implementation in tests?

Reply:
Yes! will do this once we switch over to the new implementation.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/35437

Reply:
To be fair, this is already done in https://github.com/Azure/azure-sdk-for-net/pull/33040

Reply:
Discussed offline.  Key point is that in https://github.com/Azure/azure-sdk-for-net/pull/33040, the call sites are updated to address the signature changes across types.